modelgrid_aliases_nuopc.xml

```diff
@@ -188,7 +194,7 @@
     <grid name="rof">JRA025</grid>
     <grid name="ocnice">tnx1v4</grid>
   </model_grid>
-  <model_grid alias="TL319_tn21" not_compset="_CAM">
+;  <model_grid alias="TL319_tn21" not_compset="_CAM">
```

Comment:
Invalid character that will cause xmllint to fail. Please run at least one test that validates the file.